src/components/Checkbox/__tests__/Checkbox.spec.js

```diff
@@ -0,0 +1,65 @@
+import { describe, expect, it } from 'vitest';
+import { mount } from '@vue/test-utils';
+
+import Checkbox from '../Checkbox.vue';
+
+const createWrapper = (props) => {
+  return mount(Checkbox, {
+    props,
+  });
+};
+
+describe('Checkbox', () => {
+  it('should render right label', () => {
+    const wrapper = createWrapper({ textRight: 'Label Test' });
+    const label = wrapper.find('.unnnic-checkbox__label__right');
+    expect(label.text()).eq('Label Test');
+
+    wrapper.unmount();
```

Comment:
You can use Vitest's `afterEach` or `beforeEach` function to avoid repeating this `unmount`. The code inside this function will run for each test.